lang_qc/db/mlwh_schema.py

```diff
@@ -0,0 +1,548 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2023 Genome Research Ltd. All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# @author mgcam <mg8@sanger.ac.uk>
+
+from sqlalchemy import Column, Computed, DateTime, ForeignKey, Index, String, Text, text
+from sqlalchemy.dialects.mysql import BIGINT as mysqlBIGINT
+from sqlalchemy.dialects.mysql import CHAR as mysqlCHAR
+from sqlalchemy.dialects.mysql import FLOAT as mysqlFLOAT
+from sqlalchemy.dialects.mysql import INTEGER as mysqlINTEGER
+from sqlalchemy.dialects.mysql import SMALLINT as mysqlSMALLINT
+from sqlalchemy.dialects.mysql import TINYINT as mysqlTINYINT
+from sqlalchemy.dialects.mysql import VARCHAR as mysqlVARCHAR
+from sqlalchemy.orm import declarative_base, relationship
+
+Base = declarative_base()
+# metadata = Base.metadata
```

Comment:
Delete

Reply:
Thanks,  deleted in https://github.com/wtsi-npg/npg_langqc/pull/138/commits/9d48fd9808fcd4e23dddc5c6d4c6a6e5385b2979